UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API/Filters/BespokeFilterAttribute.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace UKHO.ExchangeSetService.API.Filters
+{
+    /// <summary>
+    /// Authorization to allow only UKHO people to create unencrypted exchange set.
+    /// </summary>
+    public class BespokeFilterAttribute : ActionFilterAttribute
```

Comment:
@harshal11869 can we rename the class name to ExchangeSetAuthorizationFilterAttribute  ?

Reply:
Yes, To be more specific we can rename the class name as **BespokeExchangeSetAuthorizationFilterAttribute**

Reply:
File name changed from **BespokeFilterAttribute** to **BespokeExchangeSetAuthorizationFilterAttribute**